File: backend/internal/core/domain/profil_sekolah/profil_sekolah.go
package profil_sekolah

import "time"

type IDProfil int

type ProfilSekolah struct {
	IDProfil      IDProfil
	EmailSekolah  string
	NoTelpSekolah string
	KepalaSekolah string
	WakaSekolah   string
	NamaSekolah   string
	AlamatSekolah string
	LogoSekolah   *string
	CreatedAt     time.Time
	UpdatedAt     time.Time
}

File: frontend/src/types/ProfilSekolah/ProfilSekolah.ts
export type ProfilSekolahFormValues = {
  nama_sekolah: string;
  alamat_sekolah: string;
  no_telp_sekolah: string;
  email_sekolah: string;
  kepala_sekolah: string;
  waka_sekolah: string;
  logo_sekolah: File | null;
};

export type ProfilSekolahUpdatePayload = {
  email_sekolah: string;
  no_telp_sekolah: string;
  kepala_sekolah: string;
  waka_sekolah: string;
  nama_sekolah: string;
  alamat_sekolah: string;
};

export type ProfilSekolahResponse = ProfilSekolahUpdatePayload & {
  id_profil: number;
  logo_sekolah: string | null;
  created_at: string;
  updated_at: string;
};
